clang/cmake/caches/Fuchsia-stage2.cmake

```diff
@@ -32,6 +32,8 @@ set(LLVM_USE_RELATIVE_PATHS_IN_FILES ON CACHE BOOL "")
 set(LLDB_ENABLE_CURSES OFF CACHE BOOL "")
 set(LLDB_ENABLE_LIBEDIT OFF CACHE BOOL "")
 
+set(LLVM_NO_BAREMETAL_RUNTIMES OFF CACHE BOOL "")
```

Comment:
This is not an official LLVM build flag so I'd avoid `LLVM_` prefix, rather I'd call it something like `FUCHSIA_INCLUDE_BAREMETAL_RUNTIMES` with `ON` as the default value.

Reply:
I updated the change. PTAL.